### PR TITLE
Fix Xcode 10 metal compile error

### PR DESCRIPTION
**Pull request by ruslo**
@tqchen Please review

### CODE REVIEW

**Comment by ruslo** [comment]
The similar fix can be found here: https://github.com/gfx-rs/gfx/pull/2379

File: python/tvm/contrib/xcode.py [hunk 1]
```diff
@@ -126,8 +126,13 @@ def compile_metal(code, path_target=None, sdk="macosx"):
         out_file.write(code)
     file_target = path_target if path_target else temp_target
 
+    # See:
+    # - https://developer.apple.com/documentation/metal/gpu_functions_libraries/building_a_library_with_metal_s_command-line_tools#overview
```
Comment (ruslo) [inline python/tvm/contrib/xcode.py]: > ************* Module tvm.contrib.xcode
> C:130, 0: Line too long (139/100) (line-too-long)

Can we suppress this warning? I think having the URL here is important.

Comment (tqchen) [reply python/tvm/contrib/xcode.py]: sure, add a pylint suppression.

Comment (ruslo) [reply python/tvm/contrib/xcode.py]: Looks good now.

**Comment by tqchen** [comment]
Thanks @ruslo !

**Pull request merged**